docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to the Rust Exercism Track
+
+This document is a work-in-progress.
+Current maintainers are primarily preparing for the v3 launch of the Rust track.
+Nonetheless, feel free to peruse what we have written thus far!
+
+*Contributions welcome :)*
+
+Thank you so much for contributing! :tada:
+As one of the many tracks in Exercism, contributions here should observe Exercism standards like the [Code of Conduct](https://exercism.io/code-of-conduct).
+This document introduces the ways you can help and what maintainers expect of contributors.
+
+## Ways to Contribute
+
+As with many Open Source Projects, work abounds.
```

Comment:
```suggestion
As with many Open Source projects, work abounds.
```

I don't think that "Projects" deserves capitalization here; it's not really part of the phrase.